frontend/src/lib/components/Users/UserModal.svelte

```diff
@@ -86,6 +87,10 @@
         <h3>{$t('admin_dashboard.user_details_modal.locale')}</h3>
         <p class="value">{user.localizationCode}</p>
       </div>
+      <div>
+        <h3>{$t('admin_dashboard.user_details_modal.createdBy')}</h3>
+        <p class="value">{user.createdBy?.name  ?? '-'}</p>
```

Comment:
Actually, this should be `–` rather than `-` for consistency with other fields. The difference is subtle and hard to see in some fonts, but more obvious in other fonts.

In frontend/src/lib/i18n/index.ts there's a `NULL_LABEL` constant which isn't exported; the best thing to do here would be to edit frontend/src/lib/i18n/index.ts and export `NULL_LABEL`, then reference `NULL_LABEL` here.